ui/Surface.js

```diff
@@ -191,6 +181,15 @@ Surface.Prototype = function() {
         this.clipboard.attach(el);
       }
     }
+    
+    if (this.props.overlay) {
+      el.append(
```

Comment:
This is causing trouble, because ContainerEditor relies on the order of child elements when doing incremental updates.
Having this here, introduces an offset +1. So atm the wrong components get disposed, or components get inserted at the wrong position.